package.json

```diff
@@ -1,65 +1,86 @@
 {
-  "name": "@sanity/hierarchical-document-list",
-  "version": "1.1.0",
-  "author": "Sanity <hello@sanity.io>",
+  "name": "@practical-js/sanity-hierarchical-docs",
```

Comment:
Is your intent to take ownership/maintainership responsibility of this plugin?

If so, it would be better to fork this plugin outright rather than pushing these package changes upstream 🙏 .

If not, can you revert the package name, author, repository, etc metadata changes to this file?

Reply:
done